index.js

```diff
@@ -29,5 +29,6 @@ module.exports = {
   get utils () { return require('./utils'); },
   get osgi () { return require('./osgi'); },
   get cache () { return require('./cache'); },
-  get time () { return require('./time'); }
+  get time () { return require('./time'); },
+  get Quantity () { return require('./quantities'); }
```

Comment:
Should 'Quantity' be 'quantity' here to match the other exports which are lowercase?

Reply:
I have also asked that myself, but given that the `Quantity` export is a factory function itself and the other exports aren't, I decided to make it uppercase:

```javascript
const { items, things, rules, Quantitiy } = require('openhab');

items.getItem('Kitchen_Light');
things.getThing('lightbulb')
rules.JSRule(...)

// But

Quantitiy('1.364 nm')
```

Reply:
ahh, makes sense.